modules/reference/pages/rpk/rpk-transform/rpk-transform-build.adoc

```diff
@@ -0,0 +1,43 @@
+= rpk transform build
+:description: rpk transform build
+
+Build a transform.

+

+This command looks in the current working directory for a transform.yaml file.

+

+Then depending on the language, will install the appropriate build plugin then 

+build a .wasm file.

+

+Extra arguments are passed directly to the underlying toolchain when it invoked.


```

Comment:
@micheleRP grammar

Reply:
@rockwotj: rpk commands get pulled into our doc directly from the code. If I edit our doc manually, it'll get overwritten the next time we pull the code. Can I give you edits for the `rpk transform` commands separately?

Reply:
Please!

Reply:
`rpk transform` edits are in https://github.com/redpanda-data/docs/pull/104